### ISSUE TITLE
[Accessibility]A11y_AzureDataStudio_CreateExternalTable_Keyboard:Tool tip is not accessible through keyboard.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [microsoft/azuredatastudio#5676]
**"Bug Description:**
Check out Keros! - Identify accessibility bugs before check-in and make bug fixing faster and easier.
[11:40 AM] Chitrarupa Sharma (Tata Consultancy Services Ltd)
    
GitHubTags: #A11yMAS; #A11yTCS; #MAS1.1.1; #MAS2.1.1; #A11y_AzureDataStudio; #Win32; #DesktopApp

**Environment Details:**
Application name: Azure Data Studio (Agent, Create External Data, Notebooks)
App Version: 1.8.0-insider (user setup)
VS Code 1.33.1
Electron: 3.1.8
Node.js: 10.2.0
V8: 6.6.346.32
OS: Windows_NT x64 10.0.18362
**Additional Details:**
Tool Name: keyboard
MAS Reference:MAS1.1.1,MAS2.1.1
**Repro Steps:**
1.Install the app Azure Data Studio.
2.Go to Enable SQL server2019 preview->select table from the database.
3.pres enter or space bar to open the Create External Table
4.Hover till the "select the destination database for your external table "
5.Observed the tool tip on the sensory character.

**Actual:**
tool tip on the sensory character is not accessible through key board.
![image](https://user-images.githubusercontent.com/39204909/58461282-e6220e80-814c-11e9-8776-93c80231c0f0.png)
**Similar issue has been Observed in the below pages:**
1.CreateConntionDataSrc:Enable SQL server2019 preview->select table from the database->Create External Table screen should open->Next button->Create Database Master Key screen should open->click on next button.

**Expected:**
Tool tip should be accessible through keyboard.
User Impact:
User with motor disabilities will find it difficult to navigate the page.
**Recommendation:**
Please refer the following snippet link for fixes:
https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx
**MAS Reference:MAS1.1.1,MAS2.1.1**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={d2d2051f-bdc8-4af7-8e18-38aae867e216}
**MAS2.1.1**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}

**Attachment for Reference:**
Does this issue occur when all extensions are disabled?: Yes

[5676_AccessibilityA11y_AzureDataStudio_CreateExternalTable.pptx](https://github.com/microsoft/azuredatastudio/files/3226241/5676_AccessibilityA11y_AzureDataStudio_CreateExternalTable.pptx)

**Comment by udeeshagautam** [comment]
Same as https://github.com/microsoft/azuredatastudio/issues/6730 (not by titles but see latest comments)

Since the control is common enough and not destructive, we can mitigate this with proper documentation. Also we should open the corresponding behavior bug on VSCode.

**Comment by hannahqin** [comment]
Filed with vscode as https://github.com/microsoft/vscode/issues/83245

**Comment by hannahqin** [comment]
Closing as a duplicate of #6730 per @udeeshagautam's comment above. We'll be tracking progress on external bug documentation there.
